infra/core/security/keyvault-access.bicep

```diff
@@ -0,0 +1,22 @@
+metadata description = 'Assigns an Azure Key Vault access policy.'
```

Comment:
Please diff infra/core with latest in azure-dev repo, just to make sure we haven't diverged.

Reply:
Consistent.